src/utils/DateUtils.minMax.test.ts

```diff
@@ -0,0 +1,24 @@
+import * as Dates from '../test/data/Dates';
+import { minDate, maxDate, isBetweenDatesByDay } from './DateUtils';
+
+test('min and max date work for null', () => {
+  expect(minDate([])).toBe(null);
+  expect(maxDate([])).toBe(null);
+});
+
+test('min and max date work for normal input', () => {
+  expect(minDate([Dates.JAN_1_DATE])).toStrictEqual(Dates.JAN_1_DATE);
+  expect(maxDate([Dates.JAN_1_DATE])).toStrictEqual(Dates.JAN_1_DATE);
+});
+
+test('min and max date work for normal input', () => {
+  expect(minDate([Dates.JAN_1_DATE, Dates.DEC_1_2001_DATE, Dates.DEC_1_DATE, Dates.MAY_14_2001_DATE])).toStrictEqual(Dates.JAN_1_DATE);
+  expect(minDate([Dates.DEC_1_2001_DATE, Dates.DEC_1_DATE, Dates.MAY_14_2001_DATE])).toStrictEqual(Dates.DEC_1_DATE);
+  expect(maxDate([Dates.JAN_1_DATE, Dates.DEC_1_2001_DATE, Dates.DEC_1_DATE, Dates.MAY_14_2001_DATE])).toStrictEqual(Dates.DEC_1_2001_DATE);
+});
+
+test('isBetweenDatesByDay', () => {
```

Comment:
Please move this to a separate file (or perhaps in somewhere a bit more appropriate, like dateCompare?) -- it doesn't really fit with min/max